index.html

```diff
@@ -2326,6 +2327,102 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="comment">
+			<rdef>comment</rdef>
+			<div class="role-description">
+				<p>A comment contains a remark expressing a reaction to other content within the page.</p>
```

Comment:
I also removed the "in page" part, because I agree that it's nice to be less restrictive on that. A comment could relate to content in another page, e.g. "View next 20 comments" might take the user to a new page. However, it would usually be in the same page.